features/bevragen/persoon-beperkt/naam/in-onderzoek.feature

```diff
@@ -0,0 +1,258 @@
+    | naam                            | waarde                    |
+    | geslachtsnaam (02.40)           | Maassen                   |
+    | geboortedatum (03.10)           | 19830526                  |
+    | aanduiding in onderzoek (83.10) | <aanduiding in onderzoek> |
+    | datum ingang onderzoek (83.20)  | 20020701                  |
+    Als personen wordt gezocht met de volgende parameters
+    | naam          | waarde                              |
+    | type          | ZoekMetGeslachtsnaamEnGeboortedatum |
+    | geslachtsnaam | Maassen                             |
+    | geboortedatum | 1983-05-26                          |
+    | fields        | naam                                |
+    Dan heeft de response een persoon met de volgende 'naam' gegevens
+    | naam                                         | waarde               |
+    | geslachtsnaam                                | Maassen              |
+    | volledigeNaam                                | Maassen              |
+    | inOnderzoek.voornamen                        | <voornamen io>       |
+    | inOnderzoek.adellijkeTitelPredicaat          | <titel/predicaat io> |
+    | inOnderzoek.voorvoegsel                      | <voorvoegsel io>     |
+    | inOnderzoek.geslachtsnaam                    | <geslachtsnaam io>   |
+    | inOnderzoek.voorletters                      | <voorletters io>     |
+    | inOnderzoek.volledigeNaam                    | <volledige naam io>  |
+    | inOnderzoek.datumIngangOnderzoek.type        | Datum                |
+    | inOnderzoek.datumIngangOnderzoek.datum       | 2002-07-01           |
+    | inOnderzoek.datumIngangOnderzoek.langFormaat | 1 juli 2002          |
+
+    Voorbeelden:
+    | aanduiding in onderzoek | voornamen io | titel/predicaat io | voorvoegsel io | geslachtsnaam io | voorletters io | volledige naam io | type                      |
+    | 010000                  | true         | true               | true           | true             | true           | true              | hele categorie persoon    |
+    | 010200                  | true         | true               | true           | true             | true           | true              | hele groep naam           |
+    | 010210                  | true         |                    |                |                  | true           | true              | voornamen                 |
+    | 010220                  |              | true               |                |                  |                | true              | adellijke titel/predicaat |
+    | 010230                  |              |                    | true           |                  |                | true              | voorvoegsel               |
+    | 010240                  |              |                    |                | true             |                | true              | geslachtsnaam             |
+
```

Comment:
Hier zou ik ook geslacht (010410) toevoegen. Die heb je ook toegevoegd bij naam.volledigeNaam in fields-mapping-gba-beperkt.csv. Dus als aanduiding in onderzoek=010400 of 010410, dan is volledigeNaam in onderzoek.

Reply:
toegevoegd